### PR TITLE
Enable Zero Copy for ST 2110-22 RX.

### PR DESCRIPTION
**Pull request by qiang-han**

### CODE REVIEW

**Review by jianbingzhu — APPROVED** [review]
LGTM

**Review by djie1 — CHANGES_REQUESTED** [review]
And this PR is not rebased to master branch yet.

File: media-proxy/src/mtl.c [hunk 1]
```diff
@@ -696,9 +738,6 @@ static void* rx_st20p_frame_thread(void* arg)
         st20p_rx_put_frame(handle, frame);
     }
 
-    // printf("%s(%d), stop\n", __func__, s->idx);
-    s->fb_recv++;
```
[the remark this answers is not in the document]
Comment (qiang-han) [reply media-proxy/src/mtl.c:700]: > why do you remove this count?

it's been added in "rx_st20p_consume_frame".

File: media-proxy/include/mtl.h [hunk 2]
```diff
@@ -193,7 +204,13 @@ typedef struct {
     pthread_mutex_t st22p_wake_mutex;
 
     size_t frame_size; /* Size (Bytes) of single frame. */
+    uint16_t fb_idx;
     uint32_t fb_count; /* Frame buffer count. */
+    int fb_cnt;
```
[the remark this answers is not in the document]
Comment (qiang-han) [reply media-proxy/include/mtl.h:209]: > what is the difference between "fb_count" and "fb_cnt"?

will be fixed in other PR.

**Pull request merged**